internal/commands/update_release.go

```diff
@@ -3,13 +3,13 @@ package commands
 import (
 	"fmt"
 	"github.com/pivotal-cf/kiln/pkg/cargo"
```

Comment:
This import looks to have moved last time we did something like this.  It should live with its other github.com import friends